build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -153,10 +153,15 @@ public void CrossgenDirectory(BuildTargetContext c, string pathToAssemblies)
 
                 IList<string> crossgenArgs = new List<string> {
                     "-readytorun", "-in", file, "-out", tempPathName,
-                    "-JITPath", GetLibCLRJitPathForVersion(),
                     "-platform_assemblies_paths", platformAssembliesPaths
                 };
 
+                if (CurrentPlatform.IsUnix)
+                {
+                    crossgenArgs.Add("-JITPath");
```

Comment:
If the jit binary is depended on by the Microsoft.NETCore.App Package it will be placed in the shared framework directory and in the appropriate installers.